integration/network/service_test.go

```diff
@@ -32,18 +31,8 @@ func TestServiceWithPredefinedNetwork(t *testing.T) {
 	})
 	assert.NilError(t, err)
 
-	pollSettings := func(config *poll.Settings) {
-		if runtime.GOARCH == "arm64" || runtime.GOARCH == "arm" {
-			config.Timeout = 50 * time.Second
-			config.Delay = 100 * time.Millisecond
-		} else {
-			config.Timeout = 30 * time.Second
-			config.Delay = 100 * time.Millisecond
-		}
-	}
-
 	serviceID := serviceResp.ID
-	poll.WaitOn(t, serviceRunningCount(client, serviceID, instances), pollSettings)
+	poll.WaitOn(t, serviceRunningCount(client, serviceID, instances), swarm.NetworkPoll)
```

Comment:
Many of these are waiting on  service state, so they should use ServicePoll.

Reply:
Done!